lib/git_reflow/git_helpers.rb

```diff
@@ -40,11 +40,12 @@ def fetch_destination(destination_branch)
       run_command_with_label "git fetch origin #{destination_branch}"
     end
 
-    def update_destination(destination_branch)
+    def update_destination(remote = 'origin', destination_branch)
```

Comment:
If you change the signature of the method this way, you have to update all callers as well.  There is no reason to break existing code like this though since the new parameter is optional, add as the second param.  Then callers keep working as is.